composer.json

```diff
@@ -21,25 +21,24 @@
     ],
     "require": {
         "composer/installers": "^1.2",
-        "cweagans/composer-patches": "^1.6",
-        "drupal-composer/drupal-scaffold": "^2.2",
+        "cweagans/composer-patches": "^1",
+        "drupal-composer/drupal-scaffold": "^2",
         "drupal/config_direct_save": "^1.0",
-        "drupal/config_installer": "^1.0",
-        "drupal/config_provider": "2.0-alpha1",
-        "drupal/config_readonly": "^1.0@beta",
-        "drupal/config_sync": "2.0-alpha4",
+        "drupal/config_provider": "^2",
+        "drupal/config_readonly": "^1",
+        "drupal/config_sync": "^2",
         "drupal/console": "^1",
-        "drupal/core": "~8.0",
-        "drush/drush": "~8.0|^9.0.0-beta7",
+        "drupal/core": "^8",
+        "drush/drush": "^9",
         "joachim-n/composer-manifest": "^1.0",
         "oomphinc/composer-installers-extender": "^1.1",
         "webflo/drupal-finder": "^1.0.0",
-        "webmozart/path-util": "^2.3"
+        "webmozart/path-util": "^2.3",
+        "zaporylie/composer-drupal-optimizations": "^1"
     },
     "require-dev": {
         "behat/mink": "~1.7",
         "behat/mink-goutte-driver": "~1.2",
-        "drupal/devel": "^1.2",
```

Comment:
ynot? :(

Reply:
I'm trying to make it a point that we don't include devel and people use xdebug. Why do you think we should be?

Reply:
While I'm 💯 down for pushing XDebug, I think Devel still has value. `devel_generate` is something I use frequently. Web Profiler is useful, and I think there are some occasions where `kint` still has value, like if you're trying to troubleshoot a breakpoint.